packages/zsh-history-substring-search-git/zsh-history-substring-search-git.pacscript

```diff
@@ -0,0 +1,19 @@
+name="zsh-history-substring-search-git"
+pkgname="zsh-history-substring-search"
+url="https://github.com/zsh-users/zsh-history-substring-search.git"
+pkgver="1.1.0"
+depends=("zsh")
+breaks=("${pkgname}")
+replace=("${pkgname}")
+gives=("${pkgname}")
+arch=('amd64')
```

Comment:
Is it only on amd64?

Reply:
Ah, good catch, since it’s just a script, I think it works on all platforms which have zsh